src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -309,15 +300,15 @@ private void processFiles(List<File> files) throws CheckstyleException {
 
                 // We need to catch all exceptions to put a reason failure (file name) in exception
                 throw new CheckstyleException(
-                        getLocalizedMessage("Checker.processFilesException", filePath), ex);
+                    getLocalizedMessage("Checker.processFilesException", file), ex);
```

Comment:
`toString` is equal but less overhead:

<img width="296" alt="image" src="https://github.com/user-attachments/assets/020b0b43-bd4c-4155-8e10-86c511e7c7c0" />